### PR TITLE
Issue #1163 - Convert Teredo to use templates

### PR DESCRIPTION
**Pull request by Mailkov**
IA Page: https://duck.co/ia/view/teredo

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/Teredo.pm [hunk 1]
```diff
@@ -18,11 +18,18 @@ topics 'sysadmin';
...
+            email => ['seanoftime@gmail.com','seanoftime@gmail.com'],
+            github => ["https://github.com/Mailkov", "Melchiorre Alastra"];
 
 zci answer_type => 'teredo';
 zci is_cached   => 1;
 
+# Params: server, port, client
+sub to_text {
+	return "Teredo Server IPv4: " . $_[0] . "\nNAT Public IPv4: " . $_[2]
+		. "\nClient Port: " . $_[1];
```
Comment (moollaza) [inline lib/DDG/Goodie/Teredo.pm]: @Mailkov looks like we're mixing tabs & spaces? Please use 4-space soft-tabs. If this file has mixed indentation we can leave it for now and fix all of them later in 1 pr or something, so we're not combining indentation changes with code changes :+1:

**Comment by moollaza** [comment]
deploying to beta...

**Comment by mintsoft** [comment]
@moollaza LGTM it's not exactly looking sparklin' but I think it's acceptable

**Comment by moollaza** [comment]
@Mailkov let's add a title to this IA. It's really unclear to me what exactly we're showing, or why.

I think a title like `Teredo Address Details` would help provide some context. In general it's good to have a title with the record template to describe what's being shown

**Comment by Mailkov** [comment]
@mollaza I added title!

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

**Pull request merged**